dev/ci/user-overlays/15693-gares-reverse-coercions.sh

```diff
@@ -0,0 +1,4 @@
+overlay elpi https://github.com/proux01/coq-elpi coq_15693 15693
+overlay hierarchy_builder https://github.com/gares/hierarchy-builder reverse-coercions 15693
+overlay mathcomp https://github.com/gares/math-comp reverse-coercions 15693
```

Comment:
I am not sure why there is a mathcomp overlay. The [first commit](math-comp/math-comp#6644dcdc6820aaa151e34b0cbadf666aba259a55) of the overlay is merely a test, and will not be merged as such (or in the short-term). @proux01 is the [second commit](math-comp/math-comp#126e5d10966901e627c726e104c04f406bd9436e) still necessary? If so is it backward compatible? (If it is necessary and not backward compatible it's very bad because it would mean the next version of mathcomp will be compatible only with coq 8.16)

Reply:
The overlay was merely there to test the PR, mathcomp will keep compiling without change once the current PR is merged.
The second commit was only there so that the test kept compiling when the default reversible status of coercions (not defined with `:>` in records) was changed to `no` in the current PR.

Reply:
@proux01 thanks for the information